spec/features/empty_locations_spec.rb

```diff
@@ -5,13 +5,15 @@
 require 'rails_helper'
 
 RSpec.describe "EmptyLocations", type: :feature do
-  let!(:technician) { create(:technician) }
-  let!(:scientist) { create(:scientist) }
+  let!(:tech_swipe_card_id) { "SwipeCardId:10" }
```

Comment:
you could just use the same swipe card id? I don't think there is any technical reason why you can't.

Reply:
They have to be different swipe card ids since they are being used to find a specific user and check if that user has permissions to perform an action. Swipe card's also have uniqueness validation on them.